commands/version.go

```diff
@@ -31,35 +29,32 @@ var versionCmd = &cobra.Command{
 	Long: fmt.Sprintf(`The version command returns the current clients version information.
 
 This currently consists of the GitSHA from which the client was built.
-- https://github.com/openfaas/faas-cli/tree/%s`, GitCommit),
+- https://github.com/openfaas/faas-cli/tree/%s`, version.GitCommit),
 	Example: `  faas-cli version
   faas-cli version --short-version`,
 	Run: runVersion,
 }
 
 func runVersion(cmd *cobra.Command, args []string) {
-	if len(Version) == 0 {
-		Version = "dev"
-	}
 
 	if shortVersion {
-		fmt.Println(Version)
+		fmt.Println(version.VersionStr())
```

Comment:
There is no need for the suffix Str().

Reply:
the prefix version seemed redundant and there is some logic extracted from `commands/version.go`

Do you have an alternative name suggestion?

Reply:
Can I use `BuildVersion()`? It perfectly fits the behaviour and intent